gitops/applications/base/rook-ceph/storageclass.yaml

```diff
@@ -0,0 +1,60 @@
+# Object storage
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ceph-bucket
+provisioner: rook-ceph.ceph.rook.io/bucket
+reclaimPolicy: Delete
+volumeBindingMode: "Immediate"
+parameters:
+  objectStoreName: ceph-objectstore
+  objectStoreNamespace: "${ARGOCD_ENV_rook_ceph_namespace}"
+  region: "${ARGOCD_ENV_rook_ceph_object_store_region}"
```

Comment:
Still using the default since most  object storage claims like in gitlab and monitoring were hard coded. I just added in case we will need to remove hard coding in future and reference in argo env.

Reply:
not understanding the answer, why is region in this config?